flow/test/test_helper.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eoux pipefail
 
-cd "$(dirname $(readlink -f $0))/../"
+cd "$(dirname "$(readlink -f "$0")")/../"
 
```

Comment:
split into two commits: one for fixes, one for features, for simpler review.

Reply:
Commit has been splitted